packages/core/src/Table/ExpandableTableRow.js

```diff
@@ -0,0 +1,63 @@
+import propTypes from '@dhis2/prop-types'
+import React, { useState } from 'react'
+import { ExpandToggleCell } from './ExpandableTableRow/ExpandToggleCell.js'
+import { TableRow } from './TableRow.js'
+
+/**
+ * @module
+ * @param {TableRow.PropTypes} props
+ * @returns {React.Component}
+ * @example import { TableRow } from '@dhis2/ui'
+ * @see Live demo: {@link /demo/?path=/story/table--expandable-table-rows|Storybook}
+ */
+export const ExpandableTableRow = ({
+    children,
+    className,
+    dataTest,
+    draggable,
+    expandableContent,
+    selected,
+}) => {
+    const [isExpanded, setIsExpanded] = useState(false)
+    const toggleExpandableContent = () => setIsExpanded(!isExpanded)
```

Comment:
If we want to make this a "proper" building block, the state and callback should come from the outside, I think (also, in case the app needs to know this state, it would have direct access if it also provides the value)

Reply:
I'm not sure I fully agree with this, also see my comment here https://github.com/dhis2/ui/pull/463#discussion_r576308319. We already have the `TableRow` which is a proper building block and is able to show `expandableContent`. We could make this one fully props-driven as well, which would require us to expose three props i.e. `expandableContent`, `showExpandableContent`, and `onExpandHandleClick`. At this point I'd argue it doesn't there is very little added value for a component like this and I'd rather just remove it.

(I do see your point that this is not a true building block because it has internal state though)

Reply:
Fair enough, creating a custom one should be fairly trivial.